build/__dotNetCoreRunTimeVersions.sh

```diff
@@ -6,3 +6,5 @@ NET_CORE_APP_20='2.0.9'
 NET_CORE_APP_21='2.1.13'
 NET_CORE_APP_22='2.2.7'
 NET_CORE_APP_30='3.0.0'
+NET_CORE_APP_31='3.1.0'
```

Comment:
This was released today and is going to be LTS version in future, so thought it would be better to have some coverage for it.

Reply:
@kichalla since this is going to be LTS, should we add it to the slim build? I know that we have an existing item to add 3.0 to the slim image, should we change that to be 3.1 instead?

Reply:
Since this is still `Preview1`, we should wait until it is finally released(which will happen in December) for us to move it to slim image.